TestingConsole/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿//using BackendAPI.Repository;
+using BackendAPI.Repository.Interfaces;
+using BackendAPI.Repository.Repositories;
+using System;
+
+namespace TestingConsole
+{
+    class Program
+    {
+        static void Main(string[] args)
```

Comment:
Konsolka spoko, ale dorob tez testy, prosze.

Reply:
Testów nie da się zrobić dopóki nie będzie połączenia z bazą danych. 
Ewentualnie można by było testować mockowane metody, ale wydaje mi się że nie ma to sensu.